File: Daily Challanges/2026-07-July/2026-07-21.js
function blendWords(word1, word2) {
  const firstHalf = Math.floor(word1.length / 2);
  const secondHalf = Math.floor(word2.length / 2);
  return word1.slice(0, firstHalf) + word2.slice(secondHalf);
}

console.log(blendWords("turtle", "toucan"));
console.log(blendWords("chipmunk", "flamingo"));
console.log(blendWords("falcon", "pelican"));
console.log(blendWords("hyena", "iguana"));
console.log(blendWords("scorpion", "gorilla"));
console.log(blendWords("platypus", "wolverine"));

File: Daily Challanges/2026-07-July/2026-07-21.py
def blend_words(word1, word2):
    first_half = len(word1) // 2
    second_half = len(word2) // 2
    return word1[:first_half] + word2[second_half:]


print(blend_words("turtle", "toucan"))
print(blend_words("chipmunk", "flamingo"))
print(blend_words("falcon", "pelican"))
print(blend_words("hyena", "iguana"))
print(blend_words("scorpion", "gorilla"))
print(blend_words("platypus", "wolverine"))
